playbooks/roles/ocp-e2e/tasks/main.yml

```
@@ -69,7 +69,7 @@
         ansible_distribution == 'Ubuntu'
 
 - name: Run make target
-  make:
+  community.general.make:
```

@varad-ahirwadkar can you confirm this change is needed?
Last time when you encountered this just `collection install` was enough?

---

Yes, only `make` is sufficient.
```suggestion
  make:
```

---

Ok, since the Note on "community.general.make" is already provided in main README file, I will be closing this PR.
Thanks.